>From ec2ee6e10e3bcdf00c2e93eed427cc8c2b626cac Mon Sep 17 00:00:00 2001
From: Erik van Pienbroek <epienbro@fedoraproject.org>
Date: Sun, 19 Jan 2014 17:11:40 +0100
Subject: [PATCH 1/2] Add _gmtime32 and _localtime32 to lib64/msvcrt.def

These symbols are required by the secapi wrappers
_gmtime32_s and _localtime32_s
---
 mingw-w64-crt/lib64/msvcrt.def.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mingw-w64-crt/lib64/msvcrt.def.in b/mingw-w64-crt/lib64/msvcrt.def.in
index ed81eff..a0d3647 100644
--- a/mingw-w64-crt/lib64/msvcrt.def.in
+++ b/mingw-w64-crt/lib64/msvcrt.def.in
@@ -356,6 +356,7 @@ _getw
 _getwch
 _getwche
 _getws
+_gmtime32
 ; _gmtime32_s replaced by emu
 _gmtime64
 ; _gmtime64_s replaced by emu
@@ -471,6 +472,7 @@ _lfind
 _loaddll
 _lfind_s
 _local_unwind
+_localtime32
 ; _localtime32_s replaced by emu
 _localtime64
 ; _localtime64_s replaced by emu
-- 
1.8.4.2

